source/tyr/tyr/helper.py

```diff
@@ -229,7 +250,9 @@ def load_instance_config(instance_name):
     instance.pg_dbname = config['database']['dbname']
     instance.pg_username = config['database']['username']
     instance.pg_password = config['database']['password']
-    instance.pg_port = config['database']['port']
+    instance.pg_port = int(config['database']['port'])
+    if force_create_paths:
```

Comment:
why not always creating the paths if they do not exist ?

Reply:
this setting is there only for testing i can make a mock for `os.makedirs`
I make the change

Reply:
Done